### PR TITLE
Building a Tiny JS World

### PR DESCRIPTION
**Pull request by maxovsanyuk**
### Building a Tiny JS World

[Demo](https://maxovsanyuk.github.io/a-tiny-JS-world/)
[Code base](https://github.com/maxovsanyuk/a-tiny-JS-world/blob/master/index.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by maxovsanyuk** [comment]
Ok, thank you for advice. I to do programming less than half a year so all staffs are basically new for me. Thanks for the support. You are cool!
I hope I've removed  all files right

**Comment by OleksiyRudenko** [comment]
@maxovsanyuk Not directly related to this task, but...

1. Thank you for cleaning up the PR.
1. Please, avoid force pushing to remotes. This rewrites history and also may confuse your collaborators and contributors.
1. Please, add `.gitignore` and list files and dirs like `.DS_Store` there to have them ignored and not pushed to remotes.
1. Please, remove `.DS_Store` files/dirs (use `git rm -r`).

Sorry for the chores above, you will benefit from this practice in future.

**Comment by maxovsanyuk** [comment]
So I hope everything  is ok =)

**Comment by OleksiyRudenko** [comment]
@maxovsanyuk Nice try :) However, it seems like your amendment didn't achieve the targets.
Can you, please, reproduce the instructions [here](https://github.com/kottans/frontend-2019-homeworks/pull/37#issuecomment-442457310) one by one?

Please, note that anything specified in `.gitignore` will be (a) **ignored** (b) **after** addition to the exclusions rules. That means that if a file is already tracked adding it to `.gitignore` will not remove the file, just future amendments will be ignored.

Also `.DS_Store` is something no one wants to see in public repo as this is specific to your IDE and other people may use different IDEs.

Should you require any further clarifications, please, let me know.

File: submissions/maxovsanyuk/a-tiny-JS-world/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+```diff
++Document Object Model+
+```
+</h2>
```
Comment (OleksiyRudenko) [inline submissions/maxovsanyuk/a-tiny-JS-world/README.md]: Seems like it is a remnant from a different file not related to a Tiny JS World assignment.
Would you remove it?

**Comment by OleksiyRudenko** [comment]
I also noticed that the most of your commits say `mend`. Can you please post here an example of a command you use to commit changes?

**Comment by maxovsanyuk** [comment]
I use git commit --amend

**Comment by maxovsanyuk** [comment]
I hope I've fixed it)

**Comment by OleksiyRudenko** [comment]
Please, take a look [here](https://github.com/kottans/frontend-2019-homeworks/pull/37/files). These are the files in your PR. There are two files `.DS_Store` did you try item 4 from the instructions above?

**Comment by OleksiyRudenko** [comment]
> I use git commit --amend

It is not recommended method as it may cause conflicts with your own remote. Only amend commits that were not pushed to origin or upstream.

Having many commits is not a problem - these can be squashed when your're finally done. Just avoid `--amend` for some time.

It is also useful to describe the nature of your action in commit message. This gives your peers what they can find in a particular commit. E.g. `Delete files .DS_Store` or `Add initial code` or `Change Inhabitant class constructor definition`. This saves time and efforts for your peer developers.

Writing meaningful commit messages is an art that makes you closer to Middle developers.

Please, use `git commit -m "Meaningful commit message"`.

**Comment by OleksiyRudenko** [comment]
@maxovsanyuk so, let's start fixing things.

Create a backup branch on the commit you are currently on: `git branch backup` for not to loose any changes.

Now we need to reset `master` so it points at the most recent commit that both `upstream` and your fork/clone share.

This repo history (latest 50 commits) is:
```
*   81f1163      (HEAD -> master, origin/master, origin/HEAD) Merge pull request #33 from o-msh/dom-api-task
|\
| * 80ca86a      Fix
| * 2110889      Fix name of function
| * ab910c1      Use event delegation
| * 492c645      Split functions of getting data from API
| * 9f20637      Fix menu generating
| * 3027cbc      Add js-dom task
* |   dbcb3aa    Merge pull request #36 from Oodmincheg/memory-pair-game
|\ \
| * | e829cd0    fix requests
| * | 73164e0    add code
* | |   65a9d78  Merge pull request #14 from linkqwd/dom-api-task_linkqwd
|\ \ \
| * | | a03abde  changing event handler to eventDelegation
| * | | a814253  changing variables names
| * | | 14c582c  code corrections, ready to review
| * | | 642df46  task-6 js-dom finished
* | | |   225de4a        Merge pull request #32 from maxovsanyuk/dom-api-task
|\ \ \ \
| |_|/ /
|/| | |
| * | | 759366a  mend
| * | | b523af5  mend
| * | | 80eec9a  mend
| * | | 6099994  mend
| * | | 01533cc  Initial commit
* | | |   4e81f45        Merge pull request #35 from Oodmincheg/js-tiny-world-2
|\ \ \ \
| * | | | be306b0        use template strings, move items into array
| * | | | 3e77113        delete redundant files for this branch
| * | | | 5093b91        add new code
| * | | | 63bf1e6        add code
| * | | | 59d6cee        add memory-pair-game files
* | | | |   5055c8b      Merge pull request #19 from kalash14/dom-api-task
|\ \ \ \ \
| * | | | | ad5ab88      Changes after mentor review v2
| * | | | | 0442fc4      refactoring code after mentor review
| * | | | | a464868      Changes after reviewing code
| * | | | | 9c2d07f      Maksym Kalashnyk added his DOM homework
* | | | | | 4e8ce9c      Do the simple responsive menu (dom js task) (#28)
| |/ / / /
|/| | | |
* | | | | 9a359b5        Js oop frogger (#29)
* | | | | e9a4422        Add A Tiny JS World by feroxes (#13)
* | | | |   ed15de9      Merge pull request #16 from MitchfFirstGit/dom-api-task
|\ \ \ \ \
| * \ \ \ \   c96e15d    Merge branch 'master' into dom-api-task
| |\ \ \ \ \
| * | | | | | 60b43cc    More prettier code
| * | | | | | 70634ee    Task DOM - completed
| | |/ / / /
| |/| | | |
* | | | | |   cb4fc97    Merge pull request #11 from feroxes/js-dom-task
|\ \ \ \ \ \
| * | | | | | 966cc70    fix to template string
| * | | | | | 6b800e1    find elements with ID
| * | | | | | 34ca607    use template literal, reduce repeated code, add switch statement
| * | | | | | e03ffd9    add files
| | |_|_|/ /
| |/| | | |
* | | | | |   7768774    Merge pull request #20 from IrynaY/dom-api-task
|\ \ \ \ \ \
| |_|_|_|_|/
|/| | | | |
| * | | | |   c5d824d    Merge branch 'dom-api-task' of github.com:IrynaY/frontend-2019-homeworks into dom-api-task
| |\ \ \ \ \
| | * \ \ \ \   9771a1d  Merge branch 'master' into dom-api-task
| | |\ \ \ \ \
| | | | |_|_|/
| | | |/| | |
| | * | | | |   6a1e56f  Merge branch 'master' into dom-api-task
| | |\ \ \ \ \
| | * \ \ \ \ \   627c608        Merge branch 'master' into dom-api-task
| | |\ \ \ \ \ \
| * | \ \ \ \ \ \   e18cce4      Merge branch 'master' into dom-api-task
```

Post your `git logg -n25` output and try to figure out the commit we are looking for.

**Comment by maxovsanyuk** [comment]
I need back up to 65a9d78 commit

**Comment by maxovsanyuk** [comment]
* 2887670        (HEAD -> master, origin/master, origin/HEAD, backup) remove .DS_Store
*   22b3dfd      Merge branch 'master' of https://github.com/maxovsanyuk/frontend-2019-homeworks
|\
| * 2509f22      mend
* | a8b294e      mend
* | def2f19      commit
|/
* 26c5fdb        mend
* 772ba20        mend
* 1a9fa29        mend
* 87c7125        Initial commit
*   65a9d78      Merge pull request #14 from linkqwd/dom-api-task_linkqwd
|\
| * a03abde      changing event handler to eventDelegation
| * a814253      changing variables names
| * 14c582c      code corrections, ready to review
| * 642df46      task-6 js-dom finished
* |   225de4a    Merge pull request #32 from maxovsanyuk/dom-api-task
|\ \
| * | 759366a    mend
| * | b523af5    mend
| * | 80eec9a    mend
| * | 6099994    mend
| * | 01533cc    Initial commit
* | |   4e81f45  Merge pull request #35 from Oodmincheg/js-tiny-world-2
|\ \ \
| * | | be306b0  use template strings, move items into array
| * | | 3e77113  delete redundant files for this branch
| * | | 5093b91  add new code
| * | | 63bf1e6  add code
(END)

**Comment by OleksiyRudenko** [comment]
Great! We have `backup`  pointer to not loose our work done.

Now we forcedly move `master` pointer to 65a9d78 and forcedly sync our local clone with `origin`.

```
git checkout 65a9d78
git branch -f master
git push --force origin
```

Complete the above. Post errors if any. On success post `git logg -n25` output.

**Comment by OleksiyRudenko** [comment]
This PR has been auto-closed due to normalization of `master` ref.
Job continued in #45 

@maxovsanyuk You did a great job worth special recognition.

Here is your achievement badge:
[Kottans-Student-Contribution-workflow%3D-mastered%20contribution%20workflow-6f42c1.svg)](https://github.com/kottans/kottans/blob/master/endorsements.md)

Please, provide me with a link to a document in your repo so I could
PR the badge code into. I need it in this thread for the purpose of cross-linking the proofs.

Congratulations on the skills level-up!

**Comment by maxovsanyuk** [comment]
Thank you so much!
https://github.com/maxovsanyuk/kottans-frontend

**Pull request closed without merging**